lib/inspec/resources/virtualization.rb

```diff
@@ -36,6 +46,14 @@ def initialize
       end
     end
 
+    def virtual_system?
```

Comment:
It would be great to update [the documentation for this resource](https://github.com/inspec/inspec/blob/master/docs/resources/virtualization.md.erb) with examples and descriptions for these new `virtual_system?` and `physical_system?` matchers.